### PR TITLE
Deeplink samples feature merging nodejs

### PR DESCRIPTION
**Pull request by Prithvi-MSFT**

### CODE REVIEW

File: samples/tab-deeplink/nodejs/README.md [hunk 1]
```diff
@@ -64,6 +64,9 @@ This sample displays how to consume SubEntity Id to [DeepLink](https://docs.micr
         npm install
         ```
```
Comment (ChetanSharma-msft) [inline samples/tab-deeplink/nodejs/README.md:66]: Let's raise another PR to remove another sample

File: samples/tab-deeplink/nodejs/pages/env.js [hunk 2]
```diff
@@ -0,0 +1,3 @@
+env = {
+    AppId: "<<App-ID>>"
```
Comment (ChetanSharma-msft) [inline samples/tab-deeplink/nodejs/pages/env.js:2]: Kindly run build pipeline as well

**Review by ChetanSharma-msft — CHANGES_REQUESTED** [review]
Added my review comment.

**Comment by Prithvi-MSFT** [comment]
Build status - All builds passed
https://github.com/OfficeDev/Microsoft-Teams-Samples/actions/runs/4200665528

**Review by ChetanSharma-msft — APPROVED** [review]
Changes looks fine, approving it.

**Review by Wajeed-msft — CHANGES_REQUESTED** [review]
Please add updated images where features are combined.

File: samples/tab-deeplink/nodejs/README.md (hunk 1 above)
Comment (Prithvi-MSFT) [reply samples/tab-deeplink/nodejs/README.md:66]: PR link to remove the sample
https://github.com/OfficeDev/Microsoft-Teams-Samples/pull/723

File: samples/tab-deeplink/nodejs/pages/env.js (hunk 2 above)
Comment (Prithvi-MSFT) [reply samples/tab-deeplink/nodejs/pages/env.js:2]: Yes build pipeline all builds passed

File: samples/tab-deeplink/nodejs/README.md [hunk 3]
```diff
@@ -1,6 +1,6 @@
 ---
 page_type: sample
-description: Microsoft Teams sample app for demonstrating deeplink from Bot chat to Tab consuming Subentity ID
+description: Microsoft teams sample which displays deeplinks for call, video, chat, navigate to app and navigate within tab pages which are supported for tab and bot.
```
Comment (ChetanSharma-msft) [inline samples/tab-deeplink/nodejs/README.md:3]: Microsoft `Teams` 
update at all the places

File: samples/tab-deeplink/nodejs/README.md [hunk 4]
```diff
@@ -15,11 +15,11 @@ urlFragment: officedev-microsoft-teams-samples-tab-deeplink-nodejs
 
 # DeepLink
 
-This sample displays how to consume SubEntity Id to [DeepLink](https://docs.microsoft.com/en-us/microsoftteams/platform/concepts/build-and-test/deep-links#deep-linking-to-your-tab) from Bot to Tab and Tab to Tab.
+Microsoft teams sample which displays deeplinks for call, video, chat, navigate to app and navigate within tab pages which are supported for tab and bot.[DeepLink](https://learn.microsoft.com/microsoftteams/platform/concepts/build-and-test/deep-links)
```
Comment (ChetanSharma-msft) [inline samples/tab-deeplink/nodejs/README.md:18]: Microsoft `Teams`

File: samples/tab-deeplink/nodejs/assets/sample.json [hunk 5]
```diff
@@ -3,10 +3,10 @@
     "name": "officedev-microsoft-teams-samples-tab-deeplink-nodejs",
     "source": "officeDev",
     "title": "DeepLink",
-    "shortDescription": "Microsoft Teams sample app for demonstrating deeplink from Bot chat to Tab consuming Subentity ID",
+    "shortDescription": "Microsoft Teams sample which displays deeplinks for call, video, chat, navigate to app and navigate within tab pages which are supported for tab and bot.",
     "url": "https://github.com/OfficeDev/Microsoft-Teams-Samples/tree/main/samples/tab-deeplink/nodejs",
     "longDescription": [
-      "Microsoft Teams sample app for demonstrating deeplink from Bot chat to Tab consuming Subentity ID"
+      "Microsoft teams sample which displays deeplinks for call, video, chat, navigate to app and navigate within tab pages which are supported for tab and bot."
```
Comment (ChetanSharma-msft) [inline samples/tab-deeplink/nodejs/assets/sample.json:9]: Microsoft `Teams`

**Review by ChetanSharma-msft — CHANGES_REQUESTED** [review]
Added minor comments

File: samples/tab-deeplink/nodejs/README.md (hunk 3 above)
Comment (Prithvi-MSFT) [reply samples/tab-deeplink/nodejs/README.md]: changed

File: samples/tab-deeplink/nodejs/assets/sample.json (hunk 5 above)
Comment (Prithvi-MSFT) [reply samples/tab-deeplink/nodejs/assets/sample.json]: changed

File: samples/tab-deeplink/nodejs/README.md (hunk 4 above)
Comment (Prithvi-MSFT) [reply samples/tab-deeplink/nodejs/README.md]: changed

**Review by ChetanSharma-msft — APPROVED** [review]
Changes looks fine, approving it.

**Review by Wajeed-msft — APPROVED** [review]

**Pull request merged**